Dockerfile

```diff
@@ -0,0 +1,18 @@
+# Dockerfile
+
+FROM circleci/python:3.9.2
+
+ENV PYTHONDONTWRITEBYTECODE 1
+ENV PYTHONUNBUFFERED 1
+
+WORKDIR /code
+
+# requirements
+COPY poetry.lock pyproject.toml /code/
+
+# Create environment
+RUN poetry config virtualenvs.create false && poetry install
+RUN pip install nox
+RUN pip install nox_poetry
+
+# COPY . /code
```

Comment:
This line is redundant if we use docker-compose with 
```
    volumes:
      - .:/code
```
, because files are shared between host and container. Running server or nox will work without any problem. Should I remove it?

Reply:
Hi, I think it is not as redundant as you think. The result could be similar, but the purpose of `volume` and `ADD/ COPY` is quite different: https://docs.docker.com/compose/faq/#should-i-include-my-code-with-copyadd-or-a-volume. That is why i think we should use both.

Reply:
Done